doc/source/user_guide/materials.rst

```diff
@@ -53,9 +53,34 @@ Use settings objects
 --------------------
 This example shows how you use :ref:`ref_settings` to define materials.
 
+Copy material from database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 **Python code**
 
 .. code:: python
 
     solver.setup.materials.copy_database_material_by_name(type="fluid", name="water-liquid")
     solver.setup.cell_zone_conditions.fluid["elbow-fluid"].material = "water-liquid"
+
+Create new material
+~~~~~~~~~~~~~~~~~~~
+
+**Python code**
+
+.. code:: python
+
+    solver.setup.materials.solid["mysolid"] = {}
```

Comment:
@ansjsia 
```
>>> mysolid = solver.setup.materials.solid.create("mysolid")
>>> mysolid.chemical_formula = "SiO2"
```
Note the above create syntax. Regardless of the means of creation, I prefer not to repeat the trainwreck over multiple lines. I am referring to duplication of solver.setup.materials.solid["mysolid"]. I know that this is what a lot of authors do, but this creates a negative impression, and we should change a lot of existing test and example code.

I tried also your dict style access. It only works for already existing materials. It does not create. It raises an exception if the material does not exist.

Reply:
Thanks for the comments, I have revised the syntax in the following commit. Please let me know if further changes are required.